core/src/main/java/com/orange/ouds/core/theme/OudsGrids.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.core.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.orange.ouds.theme.OudsAdaptiveTokenValue
+import com.orange.ouds.theme.OudsAdaptiveWindowType
+import com.orange.ouds.theme.OudsGridKeyToken
+import com.orange.ouds.theme.currentWindowWidth
+import com.orange.ouds.theme.tokens.semantic.OudsGridSemanticTokens
+
+data class OudsGrids(
```

Comment:
We sometimes use plural (`OudsGrids` and `OudsBorders`) and sometimes singular (`OudsElevation` and `OudsOpacity`), which is not consistent. Maybe plural should be the correct one to use?